securedrop/tests/migrations/migration_3d91d6948753.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+import random
+import uuid
+
+from sqlalchemy import text
+from sqlalchemy.exc import NoSuchColumnError
+
+from db import db
+from journalist_app import create_app
+from models import Source
+from .helpers import random_bool, random_chars, random_datetime, bool_or_none
+
+random.seed('ᕕ( ᐛ )ᕗ')
+
+
+class UpgradeTester():
+
+    '''This migration verifies that the UUID column now exists, and that
+    the data migration completed successfully.
+    '''
+
+    SOURCE_NUM = 200
+
+    def __init__(self, config):
+        self.config = config
+        self.app = create_app(config)
+
+    def load_data(self):
+        with self.app.app_context():
+
+            for _ in range(self.SOURCE_NUM):
```

Comment:
My suspicion about `enumerate` in the migrations could be addressed here by using `range(self.SOURCE_NUM, 2)`.